lib/ts/recipe/session/sessionClass.ts

```diff
@@ -167,7 +167,9 @@ export default class Session implements SessionContainerInterface {
                     this.accessToken,
                     this.frontToken,
                     this.helpers.config,
-                    this.reqResInfo.transferMethod
+                    this.reqResInfo.transferMethod,
+                    this.reqResInfo.req,
+                    {}
```

Comment:
why passing in {}? If the user has provided a userContext, pass that in, else create a default user context (like it's done in the apis).